code/game/objects/effects/decals/Cleanable/misc.dm

```diff
@@ -331,3 +331,47 @@
 
 /obj/effect/decal/cleanable/virusdish/persistent/post_mapsave2atom(var/list/L)
 	icon_state = "brokendish-persistent"
+
+
+/obj/effect/decal/cleanable/salt
+	name = "salt"
+	desc = "Guaranteed to ward off ghouls, ghosts, geists, and low blood pressure."
+	gender = PLURAL
+	reagent = SODIUMCHLORIDE
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "salt"
+	anchored = 1
```

Comment:
I guess this is fine to prevent dragging salt but it's also weird because it means wind can't blow around salt

Reply:
I think it would be weirder to drag around a pile of salt. Ash is anchored as well.